src/Inviqa/Emarsys/Api/Response/ContactResponse.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Inviqa\Emarsys\Api\Response;
+
+class ContactResponse
+{
+    private $isSuccessful;
+
+    /**
+     * @throws \LogicException
+     */
+    public static function fromClientResponse(ClientResponse $response): self
```

Comment:
Why did you create a named constructor if there is only one?

Reply:
This follows the same pattern as the other responses